tests/unit/test-loader-test.js

```diff
@@ -1,9 +1,15 @@
-/* globals QUnit */
+/* globals QUnit, require */
 import { module, test } from 'qunit';
 
+var testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
+
+if (Object.keys(requirejs.entries).indexOf(testLoaderModulePath) === -1) {
```

Comment:
Yeah, I first thought `requirejs.entries` was an array, so this is a mistake from iterating on that idea. Will fix.